docs/ABI/Mangling.rst

```diff
@@ -217,6 +217,7 @@ types where the metadata itself has unknown layout.)
   global ::= global 'To'                 // swift-as-ObjC thunk
   global ::= global 'TD'                 // dynamic dispatch thunk
   global ::= global 'Td'                 // direct method reference thunk
+  global ::= global 'TE'                 // distributed actor thunk
```

Comment:
okey E then! Thanks for spotting this

Reply:
Hmmm we also used the `Td` in:

```

  case Kind::DistributedThunkAsyncFunctionPointer: {
    std::string Result = getSILDeclRef().mangle();
    Result.append("Td");
    Result.append("Tu");
    return Result;
  }

```

which this doesnt fix yet -- i think there we also need to use TE right? I'll follow up with this.